pulp_ansible/tests/functional/api/collection/v2/test_sync.py

```diff
@@ -14,13 +14,14 @@
 from pulp_smash.pulp3.bindings import monitor_task, delete_orphans
 
 
+# TODO: Rewrite or delete?
```

Comment:
So you are saying we might consider removing the v2 tests completely instead of rewriting them in pytest style?

Reply:
Pretty much, considering the new galaxy doesn't support the v2 api anymore, it might not be worth it to keep the tests around if v2 isn't used anywhere.